.github/workflows/build.yml

```
@@ -7,52 +7,93 @@ on:
-        file: Dockerfile
-        sbom: true
-        build-args: |
-          VERSION=${{ github.ref_name }}
+      - name: Checkout code
+        uses: actions/checkout@v4
+
+      - name: Build and publish image
+        uses: rancher/ecm-distro-tools/actions/publish-image@master
+        with:
+          image: backing-image-manager
+          tag: ${{ needs.build.outputs.image_tag }}
```

actionlint (1.7.4)

property "build" is not defined in object type {build_info: {outputs: {image_tag: string; version_build: string; version_major: string; version_minor: string}; result: string}}

(expression)